.ci/buildDockerImages.groovy

```diff
@@ -133,6 +98,42 @@ pipeline {
         }
       }
     }
+    stage('Build agent Node.js images'){
+      options {
+        skipDefaultCheckout()
+        warnError('Build agent Node.js images failed')
+      }
+      when{
+        beforeAgent true
+        expression { return params.nodejs }
+      }
+      steps {
+        dir('apm-agent-nodejs'){
+          git 'https://github.com/elastic/apm-agent-nodejs.git'
+          script {
+            def nodeVersions = readYaml(file: '.ci/.jenkins_nodejs.yml')['NODEJS_VERSION']
+            def tasks = [:]
+            nodeVersions.each { version ->
+              // Versions are double quoted
+              def nodejsVersion = version.replaceFirst('"', '')
+              tasks["${version}"] = {
+                node('ubuntu-18 && immutable && docker'){
+                  dockerLoginElasticRegistry()
+                  buildDockerImage(
+                    repo: 'https://github.com/elastic/apm-agent-nodejs.git',
+                    tag: 'apm-agent-nodejs',
+                    version: "${nodejsVersion}",
+                    folder: '.ci/docker/node-container',
+                    options: "--build-arg NODE_VERSION='${nodejsVersion}'",
```

Comment:
I'm a bit confused by these single-quotes. Do they not interfere with the interpolation of `${nodejsVersion}`?

Reply:
Because the enclosing ones are double quotes, then it should interpolate them properly. 

OK: `"blah '$var'"`
Wrong: `'blah "$var"'`

@v1v, @kuisathaverat is this correct?

Reply:
exactly

Reply:
Interesting. Thanks for the explanation!